docs/manual/common/concepts/ES_CQRS.md

```diff
@@ -1,59 +1,16 @@
-# Event Sourcing and CQRS
+# Managing data persistence
 
-Event sourcing and CQRS (Command Query Responsibility Segregation) are fundamental concepts behind Lagom's support for services that store information.
+When designing your microservices, remember that each service should own its data and have direct access to the database. Other services should then use the Service API to interact with the data. There must be no sharing of databases across different services since that would result in a too-tight coupling between the services. In this way, each microservice defines a bounded context, similar to [Bounded Contexts](http://martinfowler.com/bliki/BoundedContext.html) in Domain-Driven Design.
 
-## Why event sourcing?
+To achieve this decoupled architecture, Lagom's persistence module promotes use of [event sourcing](https://msdn.microsoft.com/en-us/library/jj591559.aspx) and [CQRS](https://msdn.microsoft.com/en-us/library/jj591573.aspx). Event sourcing is the practice of capturing all changes as domain events, which are immutable facts of things that have happened. For example, when Mike transfers money from his checking account to savings, a system using ES stores that event as "$100.00 transferred from checking to savings", rather than the complex interaction that would take place in a CRUD application, where a variety of reads and updates would need to take place before the wrapping transaction commits. 
```

Comment:
I wanted something different from "Alice" and seats because that is the example used in the MS page. I will change it to a deposit into the same account.